### PR TITLE
1349 dail pick bug

### PR DESCRIPTION
**Pull request by IlseFerris**
I tested both ADMIN scripts, the TS doesn't need testing.

### CODE REVIEW

File: admin/dail-12-month-contact.vbs
Comment (MDR22) [inline admin/dail-12-month-contact.vbs:1]: Only thought, should there be a changelog update for this script?

**Review by MDR22 — APPROVED** [review]
Looks good. Only comment related to changelog update for 12-month-contact script.

File: admin/dail-12-month-contact.vbs
Comment (IlseFerris) [reply admin/dail-12-month-contact.vbs:1]: Whoops, forgot to add this one :) 
Good catch!

Comment (IlseFerris) [reply admin/dail-12-month-contact.vbs:1]: @MDR22 fun fact, I did add the changelog, but I forgot to save it. Sigh...

**Pull request merged**